Gibe.Umbraco.Blog.Tests/BlogServiceTests.cs

```diff
@@ -1,20 +1,142 @@
 ﻿using System;
+using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
+using Examine;
+using Gibe.DittoServices;
+using Gibe.DittoServices.ModelConverters;
+using Gibe.Pager.Interfaces;
+using Gibe.Pager.Models;
+using Gibe.Pager.Services;
+using Gibe.Umbraco.Blog.Filters;
 using Gibe.Umbraco.Blog.Models;
+using Gibe.UmbracoWrappers;
 using NUnit.Framework;
+using Moq;
+using Our.Umbraco.Ditto;
+using umbraco.cms.presentation.create.controls;
+using Umbraco.Core.Models;
 using Umbraco.Core.Models.Membership;
+using Umbraco.Web.Models.TemplateQuery;
 
 namespace Gibe.Umbraco.Blog.Tests
 {
 	[TestFixture]
 	public class BlogServiceTests
 	{
+		private Mock<IPagerService> _pagerService;
+
+		[SetUp]
+		public void SetUp()
+		{
+			_pagerService = new Mock<IPagerService>();
```

Comment:
There doesn't seem to be much point to setting up a pager mock but not doing anything with it here - I find it can be advantageous to pull these things out into a `public IPagerService PagerService(whatever, parameters, are, necessary, if, any, here)` so we can encapsulate all the behaviour behind it in one place, instead of spread out through all the tests

Reply:
IPagerService is part of a different nuget package, therefore I can't edit IPagerService in this instance.

Reply:
You don't have to edit it to make a function which returns one, why would you?

Reply:
Oh, I didnt read it properly, I thought you wanted me to make a FakePagerService under IPagerService, ill do this